src/test/resources/__files/prisoner-details/GET_prisoner_details_success_any_number.json

```diff
@@ -0,0 +1,257 @@
+  "offenderId": 356,
+  "rootOffenderId": 300,
+  "firstName": "firstName",
+  "middleName": "middleName",
+  "lastName": "lastName",
+  "dateOfBirth": "1970-03-15",
+  "age": 0,
+  "activeFlag": true,
+  "facialImageId": 0,
+  "agencyId": "string",
+  "assignedLivingUnitId": 0,
+  "religion": "string",
+  "language": "string",
+  "interpreterRequired": true,
+  "writtenLanguage": "string",
+  "alertsCodes": [
+    "string"
+  ],
+  "activeAlertCount": 0,
+  "inactiveAlertCount": 0,
+  "alerts": [
+    {
+      "alertId": 1,
+      "bookingId": 14,
+      "offenderNo": "G3878UK",
+      "alertType": "X",
+      "alertTypeDescription": "Security",
+      "alertCode": "XER",
+      "alertCodeDescription": "Escape Risk",
+      "comment": "Profession lock pick.",
+      "dateCreated": "2019-08-20",
+      "dateExpires": "2020-08-20",
+      "modifiedDateTime": "2021-07-05T10:35:17",
+      "expired": true,
+      "active": false,
+      "addedByFirstName": "John",
+      "addedByLastName": "Smith",
+      "expiredByFirstName": "John",
+      "expiredByLastName": "Smith"
+    }
+  ],
+  "assignedLivingUnit": {
+    "agencyId": "string",
+    "locationId": 0,
+    "description": "string",
+    "agencyName": "string"
+  },
+  "physicalAttributes": {
+    "sexCode": "M",
+    "gender": "Male",
+    "raceCode": "W1",
+    "ethnicity": "White: Eng./Welsh/Scot./N.Irish/British",
+    "heightFeet": 5,
+    "heightInches": 60,
+    "heightMetres": 1.76,
+    "heightCentimetres": 176,
+    "weightPounds": 50,
+    "weightKilograms": 67
+  },
+  "physicalCharacteristics": [
+    {
+      "type": "string",
+      "characteristic": "string",
+      "detail": "string",
+      "imageId": 0
+    }
+  ],
+  "profileInformation": [
+    {
+      "type": "string",
+      "question": "string",
+      "resultValue": "string"
+    }
+  ],
+  "physicalMarks": [
+    {
+      "type": "string",
+      "side": "string",
+      "bodyPart": "string",
+      "orientation": "string",
+      "comment": "string",
+      "imageId": 0
+    }
+  ],
+  "assessments": [
+    {
+      "bookingId": 123456,
+      "offenderNo": "GV09876N",
+      "classificationCode": "C",
+      "classification": "Cat C",
+      "assessmentCode": "CATEGORY",
+      "assessmentDescription": "Categorisation",
+      "cellSharingAlertFlag": true,
+      "assessmentDate": "2018-02-11",
+      "nextReviewDate": "2018-02-11",
+      "approvalDate": "2018-02-11",
+      "assessmentAgencyId": "MDI",
+      "assessmentStatus": "A",
+      "assessmentSeq": 1,
+      "assessmentComment": "Comment details",
+      "assessorId": 130000,
+      "assessorUser": "NGK33Y"
+    }
+  ],
+  "csra": "string",
+  "csraClassificationCode": "STANDARD",
+  "csraClassificationDate": "2024-02-29",
+  "category": "string",
+  "categoryCode": "string",
+  "birthPlace": "WALES",
+  "birthCountryCode": "GBR",
+  "inOutStatus": "IN, OUT, TRN",
+  "identifiers": [
+    {
+      "type": "PNC",
+      "value": "1231/XX/121",
```

Comment:
Please can we have a valid PNC here? It may trip us up later
Here are some valid PNCs:
https://github.com/ministryofjustice/hmpps-person-record/blob/main/src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PNCIdentifierTest.kt#L76